system/lib/emmalloc.c

```diff
@@ -657,8 +657,6 @@ static size_t validate_alloc_alignment(size_t alignment)
   // Cannot perform allocations that are less than 4 byte aligned, because the Region
```

Comment:
Perhaps replace `4 byte` with `MALLOC_ALIGNMENT` here

Reply:
I actually think this might be accurate, as in wasm32 those data structures contain 32-bit pointers, and so they need 4-byte alignment (to avoid being unaligned). At least that's how I interpret the comment?

Reply:
Somehow the code and comments have become out of sync then.

Why not just say "Also round up to minimum outputted alignment." since that is the only thing the code is doing.  Its not specifically not handling 4-byte alignment.